examples/v2/gke/crd/api-options.yaml

```diff
@@ -0,0 +1,16 @@
+options:
+  validationOptions:
+    schemaValidation: IGNORE_WITH_WARNINGS
+  inputMappings:
```

Comment:
We have additional mappings that are less relevant for this simple example, but more relevant when you wish to update resources. 

```
      - fieldName: metadata.resourceVersion
        location: BODY
        methodMatch: ^(put|patch)$
        value: $.resource.self.metadata.resourceVersion
```

This ensures that updates apply to the expected version, so if something has edited the resource out from under DM, you'll get an error.

```
    collectionOverrides:
    - collection: /api/v1/namespaces/{namespace}/services/{name}
      options:
        inputMappings:
        - fieldName: spec.clusterIP
          location: BODY
          methodMatch: ^(put|patch)$
          value: $.resource.self.spec.clusterIP
```

This allows you to make updates to cluster resources; otherwise you'll get an error that the clusterIP cannot be changed.

Reply:
i relaized i wasn't even using the define `api-options.yaml` in the flow. 

i moved the api-options i had as well as the suggesion in your comment directly into the cluster creation yaml and python files